tm2/pkg/crypto/keys/client/update.go

```diff
@@ -0,0 +1,75 @@
+package client
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/gnolang/gno/tm2/pkg/crypto/keys"
+)
+
+type UpdateCfg struct {
+	RootCfg *BaseCfg
+
+	Force bool
+}
+
+func NewUpdateCmd(rootCfg *BaseCfg, io commands.IO) *commands.Command {
+	cfg := &UpdateCfg{
+		RootCfg: rootCfg,
+	}
+
+	return commands.NewCommand(
+		commands.Metadata{
+			Name:       "update",
```

Comment:
I'm wondering if update makes sense here.

update to me indicates that it changes more than just the password. ie. changing the key name, for instance.

I suggest we either:

- change the ShortHelp, to indicate that `update` is meant to be able to change other stuff (not necessarily in this PR)
- change the name to something like `rotate`, to indicate this command specifically changes just the password.

Reply:
I named it Update because gnokey is an interface to the Keybase API, and Update is the name in the API. Do you want to change the Keybase API?
https://github.com/gnolang/gno/blob/09b624170ee18b466c09712843d343e3b804a737/tm2/pkg/crypto/keys/types.go#L46